src/Faithlife.Tracing.AspNetCore/AspNetCoreTracing.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Globalization;
+using System.Threading;
+using Faithlife.Tracing.AspNet;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Extensions;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Faithlife.Tracing.AspNetCore
+{
+	public static class AspNetCoreTracing
+	{
+		public static void UseTracing(this IApplicationBuilder app, Action<AspNetCoreTracingSettings> configure)
```

Comment:
I've seen both conventions of taking a configuration action and taking an options instance directly. Maybe this would be more straightforward if it took an instance of `AspNetCoreTracingSettings`?

Reply:
I don't have a strong preference, although I feel like I've seen `Action<Settings>` more often. I can't really think of a concrete benefit of using it, though.